target/linux/ipq40xx/image/generic.mk

```diff
@@ -276,7 +276,29 @@ define Device/avm_fritzbox-7530
 	SOC := qcom-ipq4019
 	DEVICE_PACKAGES := fritz-caldata fritz-tffs-nand
 endef
```

Comment:
The old device section is still there, but I think it isn't needed anymore.

Reply:
You're right. Initially, I took the Fritz!3370 as a reference but then I did not include the original device section.